sdk/storage/azure_storage_blob/src/generated/clients/blob_client.rs

```diff
@@ -1387,7 +1387,7 @@ impl BlobClient {
         path = path.replace("{blobName}", &self.blob_name);
         path = path.replace("{containerName}", &self.container_name);
         url = url.join(&path)?;
-        url.query_pairs_mut().append_pair("comp", "copy");
+        // url.query_pairs_mut().append_pair("comp", "copy");
```

Comment:
This was building the wrong request URL:
`https://ruststoragedev.blob.core.windows.net/containerqu5q0qv9/destination_blob?comp=copy`

After removing this line, the correct URL:
`https://ruststoragedev.blob.core.windows.net/containerqu5q0qv9/destination_blob`